torch_frame/data/dataset.py

```diff
@@ -205,9 +207,13 @@ def _get_mapper(self, col: str) -> TensorMapper:
             return TimestampTensorMapper(
                 format=self.col_to_time_format.get(col, None))
         elif stype == torch_frame.text_embedded:
+            if isinstance(self.text_embedder_cfg, dict):
+                text_embedder_cfg = self.text_embedder_cfg[col]
+            else:
+                text_embedder_cfg = self.text_embedder_cfg
             return TextEmbeddingTensorMapper(
-                self.text_embedder_cfg.text_embedder,
-                self.text_embedder_cfg.batch_size,
+                text_embedder_cfg.text_embedder,
+                text_embedder_cfg.batch_size,
             )
```

Comment:
Following the same pattern as `col_to_sep`, can we canonicalize beforehand, so that `self.text_embedder_cfgs` is always `dict`?
Here, we just do
```python
text_embedder_cfg = self.text_embedder_cfgs[col]
return TextEmbeddingTensorMapper(text_embedder_cfg.text_embedder, text_embedder_cfg.batch_size)
```